src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImportModes.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * DataImportModes.java
+ *
+ * Copyright (C) 2009-16 by RStudio, Inc.
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+package org.rstudio.studio.client.workbench.views.environment.dataimport;
+
+public enum DataImportModes
+{
+   Csv,
```

Comment:
Note that I think this will logically become "Text" rather than "Csv" as this mode is really a front end for `read_delim` where CSV ends up being the most common sort of delimiter but we can also have tabs, fixed width, etc.